frb_rust/src/for_generated/boilerplate.rs

```diff
@@ -20,6 +21,50 @@ macro_rules! frb_generated_boilerplate {
     };
 }
 
+#[doc(hidden)]
+#[macro_export]
+macro_rules! frb_generated_wrapper_types {
+    () => {
+        #[doc(hidden)]
+        pub(crate) struct frb_wrapper<T>(T);
```

Comment:
Yeah we can do that, the rationale I had when having it frb_wrapper was that it sort of mimicked the mirror_Something and also the type is probably something the user should never import/touch, but we can't hide it since it breaks streamsink

Reply:
Anyway it is minor problem, since users should never touch it. I agree frb's internal (i.e. auto-generated and non-user-facing) naming is somehow a mixture of various styles...